lib/vector-core/src/fanout.rs

```diff
@@ -159,7 +186,7 @@ impl Fanout {
         // Nothing to send if we have no sender.
         if self.senders.is_empty() {
             trace!("No senders present.");
-            return;
+            return Ok(());
```

Comment:
```suggestion
            Ok(());
```

Can it just be like this and early return?

Reply:
The `return` statement _is_ what makes it return early. :P

Reply:
Ugh, I need my brain to work agian.